README.md

```diff
@@ -33,6 +33,11 @@ https://www.tracelabs.org/get-involved - (World)
 # InfoSec Black Friday Deals 2022
 
 ## Newsletters
+Unsupervised Learning Subscription (Annual) \
+https://danielmiessler.memberful.com/checkout?plan=53074&coupon=blackfriday \
```

Comment:
Not sure about this particular link. It looks like you need to be logged in before you can buy it so maybe just give people the link to:
`https://danielmiessler.com/`

Reply:
good call, making that update now